tests/test_web_sendfile_functional.py

```diff
@@ -16,6 +17,19 @@
     ssl = None
 
 
+HELLO_AIOHTTP = b"Hello aiohttp! :-)\n"
+
+
+@pytest.fixture(scope="module")
+def hello_txt(tmp_path_factory) -> pathlib.Path:
```

Comment:
How about getting the suffix thorough an indirect fixture request?
```suggestion
def hello_txt(request, tmp_path_factory) -> pathlib.Path:
```
It's accessible here via `request.param`. And in the test method, it'd be set through `@pytest.mark.parametrize('hello_txt', ('txt', ), indirect=('hello_txt', ))`

Alternatively, it could be provided through the fixture params in the decorator and there wouldn't be any need for the test function-level parametrization. I.e. `@pytest.fixture(params=('txt', ), scope="module")`.

It could then just return `hello_gzip` directly, not forcing the test to reconstruct the path.

Although, the fixture could be renamed to a `compressed_file_path` so it's more generic.

Reply:
Used an indirect parameter with default to return the plain text path.  That way only the tests that use compressed paths need to be touched.